### PR TITLE
New package: fswebcam-20140113

### PR DESCRIPTION
**Pull request by diogoleal**

### CODE REVIEW

File: srcpkgs/fswebcam/template [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'fswebcam'
+pkgname=fswebcam
+version=20140113
+revision=1
+build_style=gnu-configure
+makedepends="gd-devel"
+depends="gd"
```
Comment (xtraeme) [inline srcpkgs/fswebcam/template]: I really doubt this really depends on `gd` , because it's already linked to `libgd`.

Comment (diogoleal) [inline srcpkgs/fswebcam/template]: I auditioned below after creating the template:

bin [percona-toolkit] % ./fswebcam 
./fswebcam: error while loading shared libraries: libgd.so.3: cannot open shared object file: No such file or directory
bin [percona-toolkit] % xi gd

Name Action    Version           New version            Download size
gd   install   -                 2.1.1_1                - 

Size required on disk:         394KB
Free space on disk:             16GB

Do you want to continue? [Y/n] y

[*] Downloading binary packages

[*] Verifying package integrity
gd-2.1.1_1: verifying RSA signature...

[*] Running transaction tasks
gd-2.1.1_1: unpacking ...

[*] Configuring unpacked packages
gd-2.1.1_1: configuring ...
gd-2.1.1_1: installed successfully.

0 downloaded, 1 installed, 0 updated, 1 configured, 0 removed.
bin [percona-toolkit] % ./fswebcam
--- Opening /dev/video0...
Trying source module v4l2...
Error opening device: /dev/video0
open: Permission denied
Trying source module v4l1...
Error opening device: /dev/video0
open: Permission denied
Unable to find a source module that can read /dev/video0.

(xi is alias for xbps-install)

**Comment by xtraeme** [comment]
What I'm saying is that the binary is linked against the `libgd` shared library already:

https://travis-ci.org/voidlinux/void-packages/jobs/91311111#L491

So that this `depends` line is just redundant.

**Comment by diogoleal** [comment]
removed!

**Pull request closed without merging**